src/django/api/tests/test_facility_index_details_serializer.py

```diff
@@ -348,3 +353,30 @@ def test_get_other_names_embed_mode_active(self):
         expected_other_names = []
         actual_other_names = serializer.get_other_names(facility_index)
         self.assertEqual(expected_other_names, actual_other_names)
+
+    def test_get_activity_reports(self):
+        FacilityActivityReport.objects.create(
```

Comment:
Did test covered a scenario that was reproduced on this bug?

Reply:
Yes, it does.